build.gradle

```diff
@@ -140,7 +140,10 @@ intellij {
     // For a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
     version = '2024.1.3'
 
-    plugins = ['java', 'maven', 'gradle-java', 'properties', 'terminal', 'org.jetbrains.idea.maven', 'com.intellij.gradle', 'com.redhat.devtools.lsp4ij:' + lsp4ijVersion + '@nightly']
+    def lsp4ij = project.hasProperty('useLocal') && project.property('useLocal') == 'true' ?
+            file("../lsp4ij/build/distributions/LSP4IJ/") :
+            'com.redhat.devtools.lsp4ij:' + lsp4ijVersion + '@nightly'
```

Comment:
I know you didn't change this in this PR, but noticed this is hard-coded here to pull the LSP4iJ version from the `Nightly` channel. When we release we definitely want to be depending on a `Stable` release of LSP4iJ for our build and testing. I wonder if we could separate that in `build.gradle` like the `lsp4ijVersion` variable that's being read here.

Reply:
> I know you didn't change this in this PR, but noticed this is hard-coded here to pull the LSP4iJ version from the `Nightly` channel. When we release we definitely want to be depending on a `Stable` release of LSP4iJ for our build and testing. I wonder if we could separate that in `build.gradle` like the `lsp4ijVersion` variable that's being read here.

Since we have not yet decided which version of Lsp4ij to use for our release, we are currently specifying the nightly version. Once we decide to proceed with the stable release, we can remove the `@nightly` tag and specify the stable version. For now, we are using `@nightly` to test our functionalities. When we move forward with our release, we will switch to the stable version.

Reply:
@anusreelakshmi934 I think @mrglavas 's suggestion is to parameterize the build channel the same way the `lsp4ijVersion` is parameterized  - i.e. the values would be `@nightly` or `@stable`.

Reply:
@TrevCraw That is exactly what I was suggesting.

Reply:
Yes. I have updated the `lsp4ijVersion` variable at the top of build.gradle by appending `@nightly` along with the version number. If we are to use the stable version we can remove that `@nightly` and just specify the version alone.